binaries/cli/src/main.rs

```diff
@@ -536,9 +543,16 @@ fn stop_dataflow(
     let result: ControlRequestReply =
         serde_json::from_slice(&reply_raw).wrap_err("failed to parse reply")?;
     match result {
-        ControlRequestReply::DataflowStopped { uuid: _, result } => result
-            .map_err(|err| eyre::eyre!(err))
-            .wrap_err("dataflow failed"),
+        ControlRequestReply::DataflowStopped { uuid: _, result } => {
+            if result.is_ok() {
+                Ok(())
+            } else {
+                Err(eyre::eyre!(
+                    "dataflow failed: {}",
+                    FormatDataflowError(&result)
+                ))
+            }
+        }
```

Comment:
Can we make this a separate function as it repeated multiple times?

Reply:
Done in 0423155b